crates/node/src/main.rs

```diff
@@ -194,21 +192,24 @@ async fn main() -> anyhow::Result<()> {
         .with(block_provider_service)
         .with(rpc_service)
         .with(gateway_service)
-        .with(telemetry_service)
-        .with(prometheus_service);
+        .with(telemetry_service);
 
     // Check if the devnet is running with the correct chain id.
     if run_cmd.devnet && chain_config.chain_id != NetworkType::Devnet.chain_id() {
         if !run_cmd.block_production_params.override_devnet_chain_id {
-            log::error!("You're running a devnet with the network config of {:?}. This means that devnet transactions can be replayed on the actual network. Use `--network=devnet` instead. Or if this is the expected behavior please pass `--override-devnet-chain-id`", chain_config.chain_name);
+            tracing::error!("You're running a devnet with the network config of {:?}. This means that devnet transactions can be replayed on the actual network. Use `--network=devnet` instead. Or if this is the expected behavior please pass `--override-devnet-chain-id`", chain_config.chain_name);
             panic!();
         } else {
             // This log is immediately flooded with devnet accounts and so this can be missed.
             // Should we add a delay here to make this clearly visisble?
-            log::warn!("You're running a devnet with the network config of {:?}. This means that devnet transactions can be replayed on the actual network.", run_cmd.network);
+            tracing::warn!("You're running a devnet with the network config of {:?}. This means that devnet transactions can be replayed on the actual network.", run_cmd.network);
         }
     }
 
     app.start_and_drive_to_end().await?;
+
+    tracing::info!("Shutting down analytics");
```

Comment:
is it getting killed while shutting down just like other services tho?

Reply:
As mentioned in the PR description :
>Analytics doesn't implement Service like any other client, because :
>It's setup is a sync function.
>It needs to be started before anything, adding it to current service setup fails to promise that.

Analytics is not a `Service` as per's this codebase's definition is concerned.
And it needs to end at the very end to ensure that all ending logs are updated as well. 
Hence the separation in setup and shutdown